src/UI/Art/SpriteArtist/SpriteMapArtist.java

```diff
@@ -45,15 +45,15 @@ public SpriteMapArtist(GameInstance game, MapView view)
     // This nice little hack will semi-randomly decide what color fog is used for this game.
     // It switches every 10 seconds, so two games started 10 seconds apart will have different fog settings.
     String strTime = Long.toString(System.currentTimeMillis());
-    if( Integer.parseInt(new String() + (strTime.charAt(strTime.length() - 5))) % 2 == 0 )
+    if( myGame.isFogEnabled() || (Integer.parseInt(new String() + (strTime.charAt(strTime.length() - 5))) % 2 == 0) )
```

Comment:
Shouldn't this be *not* `myGame.isFogEnabled()`?

Reply:
Yes, it should. And because you are looking for context, probably. Better to have some silly comments than to miss something important.